test/functional/LoginTesterSpec.groovy

```diff
@@ -1,10 +1,14 @@
-
-
+import geb.driver.CachingDriverFactory
 import pages.CanvasPage
 import pages.LoginPage
 import geb.spock.GebReportingSpec
 
 abstract class LoginTesterSpec extends GebReportingSpec {
+
+	def setupSpec() {
+		resetBrowser()
```

Comment:
:D  what's the performance hit (in seconds total functional test running time)?

Reply:
I think it wasn't much. Maybe couple minutes at max? It's not done after every test, just every test file so the hit isn't that bad.